src/qutip_qip/compiler/circuitqedcompiler.py

```diff
@@ -226,13 +273,10 @@ def cnot_compiler(self, gate, args):
         gate1 = Gate("RX", q2, arg_value=-np.pi / 2)
         result += self.gate_compiler[gate1.name](gate1, args)
 
-        zx_coeff = self.params["zx_coeff"][q1]
-        area = 1 / 2
-        coeff, tlist = self.generate_pulse_shape(
-            args["shape"], args["num_samples"], maximum=zx_coeff, area=area
-        )
-        pulse_info = [("zx" + str(q1) + str(q2), coeff)]
-        result += [Instruction(gate, tlist, pulse_info)]
+        from ..operations.gateclass import RZX
```

Comment:
Is RZX used somewhere here?

Reply:
Oh yeah you are right I end up just using the Gate class